src/components/kompas-header-account-profile/kompas-header-account-profile.tsx

```diff
@@ -79,14 +111,17 @@ export class KompasHeaderAccountProfile {
 
             <div class="flex flex-col text-left">
               <p class="capitalize font-bold text-base text-blue-600">{truncate(this.userData?.userName, 25)}</p>
-              <p class="capitalize text-sm text-grey-600">{this.userData?.activeInfo}</p>
+              {activeInfo()}
               {expiredInfo()}
             </div>
           </div>
 
-          <div>
-            <kompas-grace-period totalGracePeriod={7} />
+          <div class="mt-4">
+            <kompas-grace-period totalGracePeriod={this.totalGracePeriod} is-coloumn={true} isShowButton={true}/>
```

Comment:
penulisan attr nya camel case aja ya
`is-column` nya diganti

Reply:
amanss mass